flink-runtime/src/main/java/org/apache/flink/runtime/rest/RestServerEndpoint.java

```diff
@@ -104,8 +105,8 @@ protected void initChannel(SocketChannel ch) {
 			}
 		};
 
-		NioEventLoopGroup bossGroup = new NioEventLoopGroup(1);
-		NioEventLoopGroup workerGroup = new NioEventLoopGroup();
+		NioEventLoopGroup bossGroup = new NioEventLoopGroup(1, new DefaultThreadFactory("flink-rest-server-netty-boss"));
+		NioEventLoopGroup workerGroup = new NioEventLoopGroup(1, new DefaultThreadFactory("flink-rest-server-netty-worker"));
 
```

Comment:
Why setting the worker group thread pool size to 1? Leaving it to 0 will give you `Runtime.getRuntime().availableProcessors() * 2`.

Reply:
ah i misread the netty source, will revert it.